src/MentionsInputSelectors.ts

```
@@ -189,32 +189,34 @@ export const getSuggestionsLayoutKey = <Extra extends Record<string, unknown>>({
     .toSorted(([left], [right]) => left - right)
     .map(
       ([childIndex, value]) =>
-        `${childIndex.toString()}|${formatQueryInfoLayoutKey(value.queryInfo)}|${value.results
-          .map(getSuggestionLayoutIdentity)
-          .join(',')}`
+        [
+          childIndex,
+          formatQueryInfoLayoutKey(value.queryInfo),
+          value.results.map(getSuggestionLayoutIdentity),
```

ESLint

[error] Do not pass function `getSuggestionLayoutIdentity` directly to `.map(…)`.

(unicorn/no-array-callback-reference)